eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/JournalStatements.scala

```diff
@@ -83,7 +88,7 @@ object JournalStatements {
               val (text, bytes) = payload match {
                 case payload: Payload.Binary => (None, Some(payload))
                 case payload: Payload.Text   => (Some(payload.value), None)
-                case payload: Payload.Json   => (Some(PlayJsonJsoniter.serializeToStr(payload.value)), None)
+                case payload: Payload.Json   => (Some(encodeTry.toStr(payload.value).get), None)
```

Comment:
Do we really call `Try.get` here or my eyes lie to me?

Reply:
Yeah, this could be fixed